src/pages/TeamOverview/TeamOverview.tsx

```diff
@@ -82,25 +93,31 @@ const TeamOverview = (): JSX.Element => {
 
     return (
         <Container>
-            <Header title={`Team ${location.state.name}`} />
-            <Search
-                originalObject={allTeamMembers}
-                updateFilteredObject={setFilteredTeamMembers}
-                placeholder='Search by name'
-                searchProps={['firstName', 'lastName', 'displayName']}
-                notifyError={setSearchError}
-            />
-            {searchError && <SearchError>{noMembersMessage}</SearchError>}
-            {!searchError && (
-                <React.Fragment>
-                    {!isLoading && showTeamLead && getTeamLeadCard(allTeamMembers[0])}
-                    <List
-                        data-testid="team-overview"
-                        isLoading={isLoading}
-                        items={getUsersCards(filteredTeamMembers.filter(member => member.id !== leadId) ?? [])}
-                    />
-                </React.Fragment>
-            )}
+            <Header title='Team'  />
+            <OverviewHeader>
+                <OverviewTitle>{location.state.name}</OverviewTitle>
+                <Search
+                    originalObject={allTeamMembers}
+                    updateFilteredObject={setFilteredTeamMembers}
+                    placeholder='Search by name'
+                    searchProps={['firstName', 'lastName']}
```

Comment:
Fixing issue on last PR, search should be only on the **name**, so `displayName` should not be a prop to search in.